lib/gceimgutils/gceremoveimg.py

```diff
@@ -148,8 +148,13 @@ def remove_images(self):
                     api.images().delete(
                         project=self.project, image=image_name
                     ).execute()
-                except HttpError:
-                    self.log.error('Unable to remove image: "%s"' % image_name)
+                except HttpError as error:
+                    msg = 'Unable to remove image: "{image}". {error}'.format(
+                        image=image_name,
+                        error=error
+                    )
+                    self.log.error(msg)
+                    raise GCERemoveImgException(msg) from error
```

Comment:
This breaks the best effort logic, i.e. if we have a number of images the code will exit on the first failure leaving other images that could potentially be deleted as "left overs". We should track the errors and collect the data and then raise after the loop if there is a problem.

Reply:
Ack, I overlooked that as I never see this function being used to delete more than a single image at a time :thinking: . Really the code is missing a `remove_image` method.